api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -392,15 +427,15 @@ function transactionDetailsReducer(
 }
 
 function coinReducer(network, amount, decimals = 6) {
-  if (!amount) {
+  if (!amount && amount !== 0) {
```

Comment:
i don't understand this... if `amount !== 0` than return an object with `amount: 0`?

Reply:
it results in NaN because it is dividing by 0